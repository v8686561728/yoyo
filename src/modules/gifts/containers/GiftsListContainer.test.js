import React from "react";
import { shallow } from "../../../enzyme";
import { shallowToJson } from "enzyme-to-json";
import { GiftsListContainer } from "./GiftsListContainer";

let props = {
  classes: {
    root: "root"
  },
  giftCards: [],
  fetchCards: jest.fn()
};
it("should render GiftsListContainer", () => {
  let wrapper = shallow(<GiftsListContainer {...props} />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});