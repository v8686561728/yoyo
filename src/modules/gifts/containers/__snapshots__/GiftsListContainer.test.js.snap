// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render GiftsListContainer 1`] = `
<WithStyles(ForwardRef(CircularProgress))
  style={
    Object {
      "marginLeft": "50%",
      "marginTop": "10%",
    }
  }
/>
`;
